Makefile

```
@@ -92,7 +92,7 @@ ifneq ($(wildcard $(ASSETS_DIR)/.),)
 	cp -r $(ASSETS_DIR) dist/$(PLUGIN_ID)/
 endif
 ifneq ($(HAS_PUBLIC),)
-	cp -r public/ dist/$(PLUGIN_ID)/
+	cp -r public/ dist/$(PLUGIN_ID)/public
```

Let's stick with the starter template

```suggestion
	cp -r public dist/$(PLUGIN_ID)/
```

---

@hanzei The code was originally putting the `icon.png` file into the `dist` folder, and not in the `dist/public` folder. This was how I was able to get the icon in the right folder.

---

You are using a mac, right? Could you please test my code?